stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -198,101 +198,101 @@ namespace {
         : Type(type), Proto(proto) {
       assert(type);
     }
+
+    friend llvm::hash_code hash_value(const ConformanceCacheKey &key) {
+      return llvm::hash_combine(key.Type, key.Proto);
+    }
   };
 
   struct ConformanceCacheEntry {
   private:
     const Metadata *Type; 
     const ProtocolDescriptor *Proto;
```

Comment:
Minor simplification?  I would just make this `const ConformanceCacheKey key`

Reply:
That is nice, and makes sure the `hash_value` implementation doesn't drift.